Sources/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -539,6 +593,67 @@ - (CGSize)getSourceDimensions {
     return CGSizeMake(0, 0);
 }
 
+- (long)findMostRecentAdvertisedBitrateForPlayerItem:(nonnull AVPlayerItem *)item {
+    AVPlayerItemAccessLog *accessLog = [item accessLog];
+    
+    if (accessLog && accessLog.events) {
+        NSArray<AVPlayerItemAccessLogEvent *> *events = [accessLog.events copy];
+        
```

Comment:
Something else we can do is try to scan this log for valid `indicatedBitrate`s, though this doesn't always work

Maybe the calculation strategy below could be a fallback instead of the default behavior.. Curious what we think here, I'm pretty comfy with either option